src/main/java/com/vaadin/recipes/recipe/contextmenuassingleselect/ContextMenuAsSingleSelect.java

```diff
@@ -0,0 +1,85 @@
+package com.vaadin.recipes.recipe.contextmenuassingleselect;
+
+
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.html.Span;
+import com.vaadin.flow.component.icon.VaadinIcon;
+import com.vaadin.flow.component.notification.Notification;
+import com.vaadin.flow.data.binder.Binder;
+import com.vaadin.flow.router.Route;
+import com.vaadin.recipes.recipe.Metadata;
+import com.vaadin.recipes.recipe.Recipe;
+
+import java.util.List;
+import java.util.StringJoiner;
+
+@Route("contextmenu-as-single-select")
+@Metadata(
```

Comment:
We would need `sourceFiles = { "SingleSelectContextMenu.java" }` to show the component source code also.

Reply:
Oh, i've overseen this..